math/big/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package big contains generic functions that operate on big.Int and can
+// be used independently of go-perun.
```

Comment:
Purge all documentation of references to Perun or `go-perun`. E.g. here, since `poly-go` is in itself already a utility package, this last sentence can just be deleted.
```suggestion
// Package big contains helpers for package math/big.
```